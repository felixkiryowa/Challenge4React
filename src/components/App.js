import React, {Component} from 'react';
// import 'bootstrap/dist/css/bootstrap.min.css';



class App extends Component {

    FormSubmit = (event) => {
        console.log("testing");
    }

    render() {
      
        return(
            <div className="container">
                <h1>Testing One Two</h1>
            </div>
        )
       
    }
}

export default App;