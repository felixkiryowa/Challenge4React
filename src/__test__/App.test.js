import React from 'react';
import {shallow} from 'enzyme';
import {App} from '../components/App';


describe('Testing App Component', () => {
    let mountApp;
    beforeEach(() => {
        mountApp = shallow(<App/>);
    });

    it('should render without crashing', () => {
        expect(mountApp).toMatchSnapshot(); 
    });
    
})