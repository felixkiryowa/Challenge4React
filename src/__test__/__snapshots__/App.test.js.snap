// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App Component should render without crashing 1`] = `
<div
  className="container"
>
  <h1>
    Testing One Two
  </h1>
</div>
`;
